### ISSUE TITLE
ssb server won't start 

### ISSUE THREAD

**Issue by aadilayub** [fraction/oasis#25]
**What version of this package are you using?**
2.9.1

**What operating system, Node.js, and npm version?**
Fedora 31, node v12.13.1, npm 6.12.1

**What happened?**
I ran `oasis --open`, a browser tab opened at `localhost:3000`, and I got a "Server Not Found" error:

![image](https://user-images.githubusercontent.com/31581044/70848365-29965d00-1e92-11ea-8cab-9c9e2857fb7a.png)

In the terminal, I got this output:

```
$ oasis --open
  oasis Initial connection attempt failed +0ms
  oasis Starting Scuttlebutt server +1ms
ssb-friends: stream legacy api used
  oasis Listening on http://localhost:3000/ +0ms
  oasis Retrying connection to own server +2s
/home/aadil/.npm-global/lib/node_modules/@fraction/oasis/src/pages/models/post.js:414
            .reduce(
             ^

TypeError: Reduce of empty array with no initial value
    at Array.reduce (<anonymous>)
    at /home/aadil/.npm-global/lib/node_modules/@fraction/oasis/src/pages/models/post.js:414:14
    at /home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/pull-stream/sinks/reduce.js:10:5
    at /home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/pull-stream/sinks/drain.js:20:24
    at /home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/pull-stream/throughs/filter.js:17:11
    at PacketStreamSubstream.weird.read (/home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/muxrpc/pull-weird.js:33:7)
    at PacketStream._onstream (/home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/packet-stream/index.js:200:12)
    at PacketStream.write (/home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/packet-stream/index.js:135:41)
    at /home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/muxrpc/pull-weird.js:56:15
    at /home/aadil/.npm-global/lib/node_modules/@fraction/oasis/node_modules/pull-stream/sinks/drain.js:24:37
```

**What did you expect to happen?**
A browser tab opens at `localhost:3000`, I can see my scuttlebutt feed

**Are you willing to submit a pull request to fix this bug?**
No

**Comment by aadilayub** [comment]
Some extra context for this bug report, I'd installed a batch of software updates from GNOME Software Center a few hours before. I'm not sure if that bears any relation to this behavior.

**Comment by aadilayub** [comment]
Okay I tried again after using Patchwork for a while and it works now...

should I close this issue?

**Comment by christianbundy** [comment]
I think this is a real bug! Recently fixed in e8b1ed910c331b54ad8ab7085d9588c9f7f17b08 though, I've just merged into `master` and released. Please let me know if you have any other trouble!
